setup.py

```diff
@@ -13,87 +13,108 @@
 
 from setuptools import find_packages, setup
 
-readme = open('README.rst').read()
-history = open('CHANGES.rst').read()
+readme = open("README.rst").read()
+history = open("CHANGES.rst").read()
+
+records_resources_version = ">=0.18.3,<0.19"
 
 tests_require = [
-    'pytest-invenio>=1.4.0',
+    "pytest-invenio>=1.4.0",
 ]
 
 extras_require = {
-    'docs': [
-        'Sphinx>=4.2.0',
+    "docs": [
+        "Sphinx>=4.2.0",
+    ],
+    "elasticsearch6": [
+        f"invenio-records-resources[elasticsearch6]{records_resources_version}"
+    ],
+    "elasticsearch7": [
+        f"invenio-records-resources[elasticsearch7]{records_resources_version}"
     ],
-    'tests': tests_require,
+    "mysql": [f"invenio-records-resources[mysql]{records_resources_version}"],
```

Comment:
I think we sort of dropped MySQL and sqlite recently. Not that we won't pick them back up, but it might clarify expectations by not including them. To be checked with @lnielsen

Reply:
From discussion: preferably keep them in actually (so good :+1: ), unless larger issue. In the future we might be able to support MySQL.